shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
```

Comment:
Do you have any documentation about the update process on modules of Prestashop? Is it "just" the function update() that needs to be implemented? I couldn't find something. Since what Prestashop version are you supporting the update of a module?